components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -348,45 +353,45 @@ end
 
 function CustomLeague:_getMaps(args)
 	local maps = {}
-	local mappages = {}
-	local index = 1
-
-	while not String.isEmpty(args['map' .. index]) do
-		-- map game mode from mapXmode
-		local mapmode = ''
-		if not String.isEmpty(args['map' .. index .. 'mode']) then
-			mapmode = MapMode.get({args['map' .. index .. 'mode']})
-		end
+	for prefix, mapInput in Table.iter.pairsByPrefix(args, 'map') do
```

Comment:
Better to use getAllForBase

Reply:
why?
afaik `|map=` never was valid, so it should be the same as using `Table.iter.pairsByPrefix`

Reply:
Alright